src/components/Skeletons/YamlView.vue

```diff
@@ -224,15 +224,24 @@ export default {
           return false
         }
 
-        // we remove the `name` and `type` because they need to be placed under `metadata`
-        delete sourceObj.type
-        delete sourceObj.name
-
         // assemble the main part of our object
         newObj.apiVersion = 'kuma.io/v1alpha1'
         newObj.kind = type
-        newObj.metadata = {
-          name: name
+        if (mesh !== undefined) { // mesh can be empty on global scoped objects
+          newObj.mesh = sourceObj.mesh
+        }
+
+        if (name.includes('.')) { // if name from Kuma has '.' it means it's k8s name joined with a namespace by dot
+          const [k8sName, namespace] = name.split('.')
```

Comment:
is it okay for multicluster where we sometimes have 2 dots in the name: `<name>.<original_ns>.default`?